apps/vessel-history/features/date-range-label/date-range-label.tsx

```diff
@@ -0,0 +1,41 @@
+import { useMemo } from 'react'
+import { useSelector } from 'react-redux'
+import { DateTime } from 'luxon'
+import cx from 'classnames'
+import { useNavigatorOnline } from '@globalfishingwatch/react-hooks'
+import { Button, ButtonType } from '@globalfishingwatch/ui-components'
+import { RISK_SUMMARY_SETTINGS } from 'data/config'
+import { Filters } from 'features/event-filters/filters.slice'
+import FiltersLabel from 'features/filters-label/filters-label'
+import { selectCurrentOfflineVessel } from 'features/vessels/offline-vessels.selectors'
+import styles from './date-range-label.module.css'
+
+export interface DateRangeLabelProps {
+  className?: string
+  type?: ButtonType
+}
+
+export function DateRangeLabel({ className, type }: DateRangeLabelProps) {
```

Comment:
why this format and not...
const DateRangeLabel: React.FC< DateRangeLabelProps > = (props): React.ReactElement => {

Reply:
It's the bolierplate generated by nx for a next component when you run:
```
nx generate @nrwl/next:component DateRangeLabel --project=vessel-history --directory=features --skipTests --no-interactive
```

Reply:
but... is what I mean is if we should keep always the same syntax